lib/validation/validator.js

```diff
@@ -1,8 +1,36 @@
 import {fileURLToPath} from "node:url";
 import {readFile} from "node:fs/promises";
 
+/**
+ * @module @ui5/project/validation/validator
+ * @description A collection of validation related APIs
+ * @public
+ */
+
+/**
+ * @enum {string}
+ * @private
```

Comment:
this is referenced from public API, so the enum must be public. Reference should be checked in JSDoc.

Reply:
So, we'd need to export it, too, right?

Reply:
After the latest refactoring, this went again to @private. The reasoning is that we now have 2 separate public functions for validation and this enum is behind their facade.